Code/Tools/AssetProcessor/native/resourcecompiler/RCBuilder.cpp

```diff
@@ -614,14 +569,6 @@ namespace AssetProcessor
         }
     }
 
-    void InternalRecognizerBasedBuilder::CreateLegacyRCJob(
-        [[maybe_unused]] const AssetBuilderSDK::CreateJobsRequest& request,
-        [[maybe_unused]] QString rcParam,
-        AssetBuilderSDK::CreateJobsResponse& response)
-    {
-        response.m_result = AssetBuilderSDK::CreateJobsResultCode::Success;
-    }
-
     bool InternalRecognizerBasedBuilder::SaveProcessJobRequestFile(const char* requestFileDir, const char* requestFileName, const AssetBuilderSDK::ProcessJobRequest& request)
```

Comment:
if you've deleted LoadProcessJobResponseFile, do we still need to SaveProcessJobRequestFiles?